azure-simple/packer/ansible-apim/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to ansible-apim
```

Comment:
@rasanjaya85 shouldn't we be adding a contributors' guide specific to this repository as well. This is actually the one from Ansible resources.

Reply:
Applies to other repositories as well.

Reply:
Done for apim